cardano-testnet/src/Parsers/Help.hs

```diff
@@ -29,12 +29,7 @@ helpAll pprefs progn rnames parserInfo = do
     go :: Parser a -> IO ()
     go p = case p of
       NilP _ -> return ()
-      OptP optP -> case optMain optP of
-        CmdReader _ cs f -> do
-          forM_ cs $ \c ->
-            forM_ (f c) $ \subParserInfo ->
-              helpAll pprefs progn (c:rnames) subParserInfo
-        _ -> return ()
+      OptP _ -> return ()
```

Comment:
Moving from version `0.16.*` to `0.18.*` of `optparse-applicative` the `CmdReader` constructor went from having three data elements to having two data elements. The data element that was removed was the middle one (`cs` above). The changes here work as if `cs` was `[]`.

Reply:
The ScopedTypeVariables for was some type annotations I sprinkled in before I heard from you about the ability to ignore the CmdReader affair, so it can go too. I'm not sure how it doesn't trip a warning of an unused feature, though.

Reply:
I did not see any warning, so we can ignore it for now.